dataset/companies/OpenAI.yaml

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
The filename should be the category the listed company belong to. So it should be something like `ai`?

Also, please capitalize alternatives. So it will be Alternatives.
I have pushed a change to this repo to make it capitalized.